src/web/mod.rs

```diff
@@ -538,11 +538,9 @@ mod test {
 
     #[test]
     fn test_index_returns_success() {
-        crate::test::with_database(|db| {
-            crate::test::with_frontend(db, |web| {
-                assert!(web.get("/").send()?.status().is_success());
-                Ok(())
-            });
+        crate::test::wrapper(|env| {
+            let web = env.frontend();
```

Comment:
Ditto for `with_frontend()`, see comments about `with_db()` above.

Reply:
Apart from this smoke test, I expect most if not all the frontend tests are going to use the database as well (for example to create dummy releases).

Reply:
We could have `with_frontend` pass a `(db, frontend)` tuple to the closure ... but the complexity might not be worth it. Up to you.

Reply:
At this point I'd prefer to keep it generic with just the `wrapper` call.